packages/cfd/src/Containers/__tests__/cfd-dashboard-container.spec.js

```diff
@@ -0,0 +1,109 @@
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(screen.getByTestId(/dt_cfd_dashboard_download_center_container/i)).toBeInTheDocument();
+    });
+    it('should render correct text according to the MT5 platform', () => {
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(
+            screen.getByText(/Run MT5 from your browser or download the MT5 app for your devices/i)
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(/The MT5 desktop app is not supported by Windows XP, Windows 2003, and Windows Vista/i)
+        ).toBeInTheDocument();
+    });
+    it('should show the proper icons for the MT5 platform ', () => {
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(screen.getByText(/IcMt5DeviceDesktop/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5DeviceLaptop/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationWindows/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationMacos/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationLinux/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5DeviceTablet/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5DevicePhone/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationGoogle/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationApple/i)).toBeInTheDocument();
+    });
+
+    it('should download/redirect the correct file for MT5 platform', () => {
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(screen.getByText(/IcInstallationWindows/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/web/deriv.limited/mt5/deriv5setup.exe'
+        );
+        expect(screen.getByText(/IcInstallationMacos/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/web/metaquotes.software.corp/mt5/MetaTrader5.dmg'
+        );
+        expect(screen.getByText(/IcInstallationLinux/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
+        );
+        expect(screen.getByText(/IcInstallationGoogle/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/mobile/mt5/android?server=Deriv-Demo,Deriv-Server'
+        );
+        expect(screen.getByText(/IcInstallationApple/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/mobile/mt5/ios?server=Deriv-Demo,Deriv-Server'
+        );
+    });
+
+    it('should render correct text according to the platform (DerivX)', () => {
+        render(<CFDDashboardContainer {...mock_props} platform='dxtrade' is_dark_mode_on='true' />);
+        expect(screen.getByText(/Run Deriv X on your browser or download the mobile app/i)).toBeInTheDocument();
```

Comment:
these two tests are not related to the `is_dark_mode_on='true'` prop, so better to move them `'should render the correct icons for the Deriv X platform'` unit

Reply:
Roger that